io/logger.go

```diff
@@ -110,7 +110,7 @@ func (l *StandardLogger) Print(data string) {
 }
 
 func (l *StandardLogger) Println(data string) {
-	_, err := fmt.Fprintln(l.stdOutFile, data)
+	_, err := fmt.Fprintln(l.stdOutFile, ""+data)
```

Comment:
note to self: these `""` additions fixes a problem where lines are printed with very strange offsets. I couldn't figure out what the issue was but this workaround was good enough